packages/amplify-e2e-tests/src/__tests__/rds-model-v2.test.ts

```diff
@@ -14,28 +14,33 @@ import {
   initJSProjectWithProfile,
   removeRDSPortInboundRule,
   getResource,
+  sleep,
 } from 'amplify-category-api-e2e-core';
 import { existsSync, readFileSync } from 'fs-extra';
 import generator from 'generate-password';
 import { ObjectTypeDefinitionNode, parse } from 'graphql';
 import path from 'path';
 import AWSAppSyncClient, { AUTH_TYPE } from 'aws-appsync';
 import gql from 'graphql-tag';
+import axios from 'axios';
 
 // to deal with bug in cognito-identity-js
 (global as any).fetch = require('node-fetch');
 
 const CDK_FUNCTION_TYPE = 'AWS::Lambda::Function';
 const CDK_VPC_ENDPOINT_TYPE = 'AWS::EC2::VPCEndpoint';
+const IPIFY_URL = 'https://api.ipify.org/';
+const AWSCHECKIP_URL = 'https://checkip.amazonaws.com/';
```

Comment:
Under what conditions would we expect these to return different values? Or are you using one as a fallback for the other?

Reply:
IPIFY Url works when I'm not connected to AWS network (or VPN). `checkip` works in AWS network. Having both to allow list enables us to run the tests from anywhere.